pkgs/tools/security/pinentry/default.nix

```
@@ -1,100 +1,96 @@
-{ fetchurl, mkDerivation, fetchpatch, stdenv, lib, pkg-config, autoreconfHook, wrapGAppsHook
-, libgpg-error, libassuan, qtbase, wrapQtAppsHook
+{ stdenv, lib, fetchurl, fetchpatch, pkg-config, autoreconfHook, wrapGAppsHook
+, libgpg-error, libassuan, libsForQt5
 , ncurses, gtk2, gcr
-, withLibsecret ? true, libsecret
-, enabledFlavors ? [ "curses" "tty" "gtk2" "emacs" ]
-  ++ lib.optionals stdenv.isLinux [ "gnome3" ]
-  ++ lib.optionals (!stdenv.isDarwin) [ "qt" ]
+, withLibcap ? !stdenv.isDarwin, libcap ? null
+, withLibsecret ? true, libsecret ? null
```

```suggestion
, withLibcap ? !stdenv.isDarwin, libcap
, withLibsecret ? true, libsecret
```
that is not required and overwriting packages with null is very discouraged